python_modules/dagster-graphql/dagster_graphql/implementation/execution/__init__.py

```diff
@@ -369,15 +371,15 @@ def wipe_assets(
     from ...schema.roots.mutation import GrapheneAssetWipeSuccess
 
     instance = graphene_info.context.instance
-    for asset_partition_range in asset_partition_ranges:
-        # This will be removed when support for partition-scoped wiping is added.
-        if asset_partition_range.partition_range is not None:
-            check.failed(
-                f"Partitioned asset wipe is not supported yet. Found partition range: {asset_partition_range}"
-            )
+    for apr in asset_partition_ranges:
+        if apr.partition_range is None:
+            instance.wipe_assets([apr.asset_key])
+        else:
+            node = check.not_none(get_external_asset_node(graphene_info, apr.asset_key))
+            partitions_def = check.not_none(node.partitions_def_data).get_partitions_definition()
+            partition_keys = partitions_def.get_partition_keys_in_range(apr.partition_range)
+            instance.wipe_asset_partitions(apr.asset_key, partition_keys)
```

Comment:
is this going to be a perf regression to have N calls to wipe assets instead of 1 call? I would suggest retaining the previous behavior for the unpartitioned assets.

Reply:
I didn't anticipate a perf regression since `DagsterInstance.wipe_assets` just loops over the keys and makes separate `EventLogStorage.wipe_asset` calls anyway, but to be conservative I've adjusted this to now retain previous behavior.